.circleci/config.yml

```diff
@@ -1,6 +1,71 @@
 version: 2.0
 jobs:
-  backup:
+  production_postgresql_backup:
+    docker:
+      - image: cimg/node:18.13.0
+    environment:
+      DEVOPS_REPO: "git@github.com:ParabolInc/action-devops.git"
+      DEVOPS_WORKDIR: "~/action-devops"
+      PRODUCTION_BACKUP_LOCATION: "dokku@backups.action-production.parabol.co"
+    working_directory: ~/action
+    resource_class: medium
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - "53:a8:37:35:c3:7e:54:f5:19:f6:8e:a1:e0:78:52:da"
+      - run:
+          name: Slack setup
+          command: |
+            curl --ssl -X POST -H 'Content-type: application/json' --data '{"text":"Starting `Production` PostgreSQL backup..."}' $SLACK_DEVOPS_URL
+      - checkout
+      - run:
+          name: DevOps checkout
+          command: |
+            git clone $DEVOPS_REPO $DEVOPS_WORKDIR
+      - run:
+          name: Backup
+          no_output_timeout: 55m
+          command: |
+            ssh -o StrictHostKeyChecking=no "${PRODUCTION_BACKUP_LOCATION}" -T >/dev/null
+            $DEVOPS_WORKDIR/postgres/postgres-backup.sh -s production
```

Comment:
-1 Why are we checking out the devops locally if we then ssh into another machine just to run the script there?

Reply:
Fixed, but to be tested.